index.html

```diff
@@ -6123,22 +6152,29 @@ <h3><dfn>Element Send Keys</dfn></h3>
   of <a>trying</a> to <a>get a known connected element</a>
   with <a>url variable</a> <var>element id</var>.
 
- <li><p><a>Scroll into view</a> the <var>element</var>.
+ <li><p>Let <var>file</var> be <code>true</code> if If <var>element</var> is
```

Comment:
We typically don’t put booleans in `<code>` tags. There are some cases we do, but that should be fixed in
https://github.com/w3c/webdriver/issues/1334. This remark applies throughout this patch.